src/AuthenticatedAPIClient/authInterface.js

```diff
@@ -100,15 +91,6 @@ export default function applyAuthInterface(httpClient, authConfig) {
       try {
         if (cookieValue) {
           decodedToken = jwtDecode(cookieValue);
-        /* istanbul ignore next */
-        } else {
-          const simplestCookieFound = global.document.cookie
```

Comment:
**Reviewer Note:** This doesn't seem to give a different result from universal-cookie.  However, just in case, I moved this to the logError code.